Type conversion error

```cpp
/home/travis/build/y2s82/goblin_camp/src/data/Config.cpp: In function ‘std::string Config::GetStringCVar(const string&)’:
/home/travis/build/y2s82/goblin_camp/src/data/Config.cpp:133:11: error: could not convert ‘'\000'’ from ‘char’ to ‘std::string’ {aka ‘std::basic_string<char>’}
    return '\0';
```

--- a/src/data/Config.cpp
+++ b/src/data/Config.cpp
@@ -130,7 +130,7 @@ namespace Config {
 	std::string GetStringCVar(const std::string& name) {
 		if (Globals::cvars.find(name) == Globals::cvars.end()) {
 			LOG("WARNING: CVar "<< name << " doesn't exist.");
-			return '\0';
+			return std::string('\0');
 		}
 		return Globals::cvars[name];
 	}

